scripts/build-api-docs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+echo "--> Fetching sentry source"
+curl https://codeload.github.com/getsentry/sentry/zip/master --silent --output sentry.zip
```

Comment:
Is this url reliable?

I'd just use the canonical https://github.com/getsentry/sentry/archive/master.zip

It'll incur a redirect, but it's not some seemingly hidden API. But maybe you know something I don't.

Reply:
No I was just skipping the redirect as I forgot about `-L` in curl earlier.